### PR TITLE
3.x added french translation to impersonate feature

### PR DESCRIPTION
**Pull request by apricotpoodle**

### CODE REVIEW

**Comment by apricotpoodle** [comment]
Add French translation of impersonating feature

File: docs/fr/impersonation.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        // Activer l'emprunt d'identité.
+        $this->Authentication->usurper l'identité($targetUser);
+        $this->redirect($this->referer());
+    }
+
+Une fois que vous avez commencé à emprunter l'identité d'un utilisateur, toutes les demandes ultérieures auront
+``$targetUser`` comme identité active.
+
+Arrêter l'emprunt d'identité
+====================
```
Comment (markstory) [inline docs/fr/impersonation.rst:46]: Title underlines need to be the same length as the text above them.

File: docs/fr/impersonation.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+        // Assurez-vous de l'activation du mode emprunt d'identité.
+        if (!$this->Authentication->isImpersonating()) {
+            throw new NotFoundException();
+        }
+        $this->Authentication->stopImpersonating();
+    }
+
+Limites à l'emprunt d'identité
+============================
```
Comment (markstory) [inline docs/fr/impersonation.rst:65]: Title underlines need to be the same length as the text above them.

**Comment by dereuromark** [comment]
ping @apricotpoodle

**Pull request closed without merging**